### DISCUSSION TITLE
Scheduling Posts with AstroJS

### DISCUSSION THREAD

**Discussion by giscus** [jonathanyeong/personal-website#36] [Blog Post Discussions]
# Scheduling Posts with AstroJS

Quick demo on how I schedule posts with AstroJS

https://jonathanyeong.com/changelog-scheduling-posts-with-astrojs/

**Comment by jonathanyeong** [comment]
I thought more about this post, and I don't think it will work as is. I forgot that the `filterPublishedPosts` tells AstroJS which posts to build. Since a post in the future won't be built, I won't automatically see a post when the date rolls around. I'm looking into triggering a rebuild of static content daily to pull in any new posts using a combination of Github Actions and Netlify Build Hooks.
